students/jacob_daylong/Lesson 3/List_Lab/List.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Sep 22 09:47:58 2020
+
+@author: jaked
+
+**Series 1**
+Create a list that contains “Apples”, “Pears”, “Oranges” and “Peaches”.
+Display the list (plain old print() is fine…).
+Ask the user for another fruit and add it to the end of the list.
+Display the list.
+Ask the user for a number and display the number back to the user and the fruit corresponding to that number (on a 1-is-first basis). Remember that Python uses zero-based indexing, so you will need to correct.
+Add another fruit to the beginning of the list using “+” and display the list.
+Add another fruit to the beginning of the list using insert() and display the list.
+Display all the fruits that begin with “P”, using a for loop.
+
+**Series 2**
+Using the list created in series 1 above:
+
+Display the list.
+Remove the last fruit from the list.
+Display the list.
+Ask the user for a fruit to delete, find it and delete it.
+(Bonus: Multiply the list times two. Keep asking until a match is found. Once found, delete all occurrences.)
+
+**Series 3**
+Again, using the list from series 1:
+
+Ask the user for input displaying a line like “Do you like apples?” for each fruit in the list (making the fruit all lowercase).
+For each “no”, delete that fruit from the list.
+For any answer that is not “yes” or “no”, prompt the user to answer with one of those two values (a while loop is good here)
+Display the list.
+
+**Series 4**
+Once more, using the list from series 1:
+
+Make a new list with the contents of the original, but with all the letters in each item reversed.
+Delete the last item of the original list. Display the original list and the copy.
+
+"""
+fruits = ["Apples", "Pears", "Oranges", "Peaches"]
+fruit_num = 0
+
+#Series_1
+print("Series 1:")
+print()
+fruits_series_1 = fruits
+
+print(fruits_series_1)
+
+new_fruit = input("Please add another fruit: ")
+
+fruits_series_1.append(new_fruit)
+
+print(fruits_series_1)
+
+fruit_num = int(input("Please select a fruit by number: "))
+
+print("User selection: " + str(fruit_num))
+print("Selected Fruit: " + fruits_series_1[fruit_num - 1])
+
+print()
+print("Add Grapes:")
+fruits_series_1  = ["Grapes"] + fruits
+print(fruits_series_1)
+
+print()
+print("Add Watermellon:")
+fruits_series_1.insert(0, "Watermellon")
+print(fruits_series_1)
+print()
+
+print("Fruits starting with 'P'")
+for i in fruits_series_1:
+    if i[0] == "P":
```

Comment:
Good idea to user upper() or something similar to catch case sensitive user input